appium/api-mocks/apis/google/google-endpoints.ts

```diff
@@ -240,6 +240,7 @@ export const getMockGoogleEndpoints = (
       } else if (isDelete(req)) {
         const id = parseResourceId(req.url!);
         const data = (await GoogleData.withInitializedData(acct, googleConfig));
+        await browser.pause(2000); // wait some time because autosave and delete mock draft api requests might be interfering
```

Comment:
This might be silly fix but maybe easiest fix :-)
I think this semaphoreci case won't happen in live situation though.
Let me know your thought @sosnovsky

Reply:
Yeah, we can try to fix it this way and monitor drafts test further, to see if it was a reason for test fail.

Reply:
It seems fine. Maybe we can merge this PR and see if this issue happens again?